src/js/__tests__/checkDocumentCSPHeaders-test.js

```diff
@@ -13,14 +13,14 @@ import {checkCSPForEvals} from '../content/checkCSPForEvals';
 import {ORIGIN_TYPE} from '../config';
 import {setCurrentOrigin} from '../content/updateCurrentState';
 
-describe('checkDocumentCSPHeaders', () => {
+describe('checkCSPForEvals', () => {
```

Comment:
This is a bit inaccurate because the `checkCSPForWorkerSrc` group is under this, right?

Reply:
True, I could create a new describe block for `checkCSPForEvals` inside `checkDocumentCSPHeaders` if that would be clearer?